CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * View affiliation associated permissions. Refs UIU-2800.
 * Confirmation modal for manual anonymization. Refs UIU-1631.
 * Fixed bug with certain accordions not closing in edit view. Refs UIU-2811.
+* Show success and Error toasts when Assign/unassign a users affiliations. Refs UIU-2852.
```

Comment:
Changelog should be updated at the end of the in progress list. Could you please correct it? i.e., swap line 26 and 27